task03/index.js

```diff
@@ -0,0 +1,121 @@
+class Human {
+    constructor(name = null, age = 0, dateOfBirth = null) {        
+        this.Name = {
```

Comment:
```suggestion
        this.name = {
```
свойство просто указывает на объект, тут тоже с маленькой буквы

Reply:
Я привык в C# так делать - все методы и свойства класса с большой буквы